go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/sirupsen/logrus v1.7.0 // indirect
 	github.com/ziutek/mymysql v1.5.4 // indirect
 	gitlab.com/yawning/obfs4.git v0.0.0-20210511220700-e330d1b7024b
+	gitlab.com/yawning/utls.git v0.0.12-1 // indirect
```

Comment:
I am surprised it's indicated as `indirect`! Would you mind running `go mod tidy` and checking whether this is enough to mark this dependency as a direct dependency?